diff --git a/build/src/start/server/metro/MetroBundlerDevServer.js b/build/src/start/server/metro/MetroBundlerDevServer.js
index ff12db2bf63605b005974645dbdbbeaa86189812..a0f5c0795b772f43897f8211e7e5a071333ffef7 100644
--- a/build/src/start/server/metro/MetroBundlerDevServer.js
+++ b/build/src/start/server/metro/MetroBundlerDevServer.js
@@ -674,9 +674,10 @@ class MetroBundlerDevServer extends _BundlerDevServer.BundlerDevServer {
             serializerOutput: 'static'
         };
         // https://github.com/facebook/metro/blob/2405f2f6c37a1b641cc379b9c733b1eff0c1c2a1/packages/metro/src/lib/parseOptionsFromUrl.js#L55-L87
-        if (!opts.mainModuleName.startsWith('/') && !_path().default.isAbsolute(opts.mainModuleName)) {
-            opts.mainModuleName = './' + opts.mainModuleName;
-        }
+        // if (!opts.mainModuleName.startsWith('/') && !_path().default.isAbsolute(opts.mainModuleName)) {
+        //     opts.mainModuleName = './' + opts.mainModuleName;
+        // }
+        opts.mainModuleName = './app'
         const output = await this.metroLoadModuleContents(opts.mainModuleName, opts, extraOptions);
         return {
             artifacts: output.artifacts,
